common/js/jquery-ui-timepicker-addon.js

```diff
@@ -251,7 +251,7 @@
 			if (o.altField) {
 				tp_inst.$altInput = $(o.altField).css({
 					cursor: 'pointer'
-				}).focus(function() {
+				}).on ( 'focus', function() {
```

Comment:
This is a library and it's being used for the `Date` field in `Editorial Metadata` module - related PR https://github.com/Automattic/Edit-Flow/pull/169
We can see it in action when editing a post. 

I see that the author has not changed this yet. I tried finding a way to test this file but it's not clear how I can replicate these events. 

https://github.com/trentrichardson/jQuery-Timepicker-Addon/blob/7160ce59c660462f4d3379951f1e141131bcb02c/src/jquery-ui-timepicker-addon.js#L287

Reply:
Hmmm yeah this is a tough one.... 

I think as long as the date picker works we're good.  I tested it on the post editing screen and it worked for me.
✅